core_api/src/routes/chat.py

```diff
@@ -1,31 +1,44 @@
 import logging
-from typing import Annotated, Any
+from typing import Annotated
 from uuid import UUID
 
 from fastapi import Depends, FastAPI, WebSocket
 from fastapi.encoders import jsonable_encoder
-from langchain_community.chat_models import ChatLiteLLM
-from langchain_core.prompts import ChatPromptTemplate
 from langchain_core.runnables import Runnable
-from langchain_elasticsearch import ElasticsearchStore
 
 from core_api.src.auth import get_user_uuid, get_ws_user_uuid
 from core_api.src.build_chains import build_retrieval_chain, build_summary_chain
-from core_api.src.dependencies import get_llm, get_storage_handler, get_vector_store
+from core_api.src.dependencies import es_retriever, llm
+from core_api.src.runnables import make_static_response_chain, map_to_chat_response
 from core_api.src.semantic_routes import (
     ABILITY_RESPONSE,
     COACH_RESPONSE,
     INFO_RESPONSE,
     route_layer,
 )
+from redbox.llm.prompts.summarisation import (
+    SUMMARISATION_QUESTION_PROMPT_TEMPLATE,
+    SUMMARISATION_SYSTEM_PROMPT_TEMPLATE,
+)
+from redbox.models.chain import ChainInput
 from redbox.models.chat import ChatRequest, ChatResponse, SourceDocument
-from redbox.storage import ElasticsearchStorageHandler
 
 # === Logging ===
 
 logging.basicConfig(level=logging.INFO)
 log = logging.getLogger()
 
+ROUTABLE_CHAINS = {
+    "info": make_static_response_chain(INFO_RESPONSE),
+    "ability": make_static_response_chain(ABILITY_RESPONSE),
+    "coach": make_static_response_chain(COACH_RESPONSE),
+    "gratitude": make_static_response_chain("You're welcome!"),
+    "retrieval": build_retrieval_chain(llm, es_retriever),
+    "summarisation": build_summary_chain(
+        llm, es_retriever, SUMMARISATION_SYSTEM_PROMPT_TEMPLATE, SUMMARISATION_QUESTION_PROMPT_TEMPLATE
+    ),
+    "extract": make_static_response_chain("You asking to extract some information - route not yet implemented"),
```

Comment:
```suggestion
    "extract": make_static_response_chain("You asking to extract some information - route not yet implemented"),
```

out of scope of this PR i know, but lets kill this off